package.json

```diff
@@ -1,7 +1,7 @@
 {
   "type": "commonjs",
   "name": "zap",
-  "version": "0.0.0",
+  "version": "2022.10.27",
```

Comment:
discard this change since the version is auto-generated during build / release time.

Reply:
@paulr34 , you do not have husky hooks properly initialized. If they are, then they do not let you commit package.json, unless the `version` field is set to "0.0.0".
Make sure you run `husky install`.

Reply:
LIke this:
```
[~/git/zap (master)]$ git commit -a
🚀   > Pretty quick reformat...
🔍  Finding changed files since git revision aa0c1150.
🎯  Found 1 changed file.
✅  Everything is awesome!
🚀   > JSdoc regen...
🚀   > Eslint...
🚀   > Reset version in package.json to 0.0.0...
⛔ Version in package.json was not set correctly. It was automatically updated to 0.0.0. Review and commit again, please.
husky - pre-commit hook exited with code 1 (error)
```

Reply:
okay thanks

Reply:
@tecimovic I think I do have husky installed because I have that output, but after I committed, I pulled, and was forced to checkout package.json for some reason. Something is wrong..

Reply:
fixed